client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -29,7 +43,7 @@ const CancelPurchaseRefundInformation = ( {
 	confirmCancelBundledDomain,
 	onCancelConfirmationStateChange,
 } ) => {
-	const { refundPeriodInDays } = purchase;
+	const { siteId, siteUrl, refundPeriodInDays } = purchase;
```

Comment:
The `Purchase` TypeScript type [defined here](https://github.com/Automattic/wp-calypso/blob/8c8de368804fc686f975e3cbd8834cd40f936f5b/packages/data-stores/src/purchases/types.ts#L3) does not have a `siteUrl` property. I assume one exists, though? If so we need to add that to the type. I'll see if I can confirm that it exists and try to add it in #94864